src/pages/themes/submit.astro

```diff
@@ -0,0 +1,105 @@
+import Base from '../../layouts/Base.astro'
+---
+
+<Base title="Submit a theme">
+    <PageTitle>
+        <Fragment slot="title">Submit a theme</Fragment>
+        Thank you for sharing your themes with the Astro community! Choose to make
+        your theme free or paid. All payments go directly to you, we don't handle
+        the sale or take a cut.
+    </PageTitle>
+
+    <section class="max-w-screen-md mx-auto px-4">
+        <h3 class="text-3xl mt-8 mb-4">Best practices</h3>
+        <p class="my-4">What we look for when reviewing submissions:</p>
+        <ul class="my-4 whitespace-normal children:before:content-['✅_']">
+            <li>Your theme uses the latest release of Astro</li>
+            <li>
+                For free themes, source code for the theme is available in a
+                public repository
+            </li>
+            <li>All NPM dependencies used in the theme are public</li>
+            <li>
+                Theme preview image or screenshot included (see below for
+                details)
+            </li>
+        </ul>
+    </section>
+
+    <form
+        method="post"
+        class="max-w-screen-sm flex flex-col items-center gap-4 mx-auto py-8 px-4"
+        data-netlify="true"
+    >
+        <InputField
+            name="authorName"
+            label="Your name"
+            placeholder="Sally Ride"
+            required
+        />
+        <InputField
+            name="authorEmail"
+            label="Your email"
+            type="email"
+            placeholder="me@website.com"
+            required
+        />
+        <InputField
+            label="Theme name"
+            name="themeName"
+            placeholder="My Awesome Theme"
+            required
+        />
+        <div class="w-fit self-start">
+            <Field label="Free">
+                <Switch name="free" checked data-free-switch />
+            </Field>
+        </div>
+        <InputField
+            label="Live demo URL"
+            name="liveDemoUrl"
+            placeholder="https://example.com/theme-demo"
+        />
+        <InputField
+            label="Public repo URL (required for free themes)"
+            name="repoUrl"
+            placeholder="https://github.com/me/my-awesome-theme"
+            data-repo-url
+        />
+        <div class="w-full">
+            <ImageInput name="previewImage" required />
+        </div>
+        <TextAreaField
+            label="Short description"
+            name="shortDescription"
+            placeholder="A short description of your theme, displayed on the theme list page"
+            required
+        />
+        <Button type="submit" theme="primary">Submit</Button>
+    </form>
+</Base>
+
+<script>
+    const freeSwitch =
+        document.querySelector<HTMLInputElement>('[data-free-switch]')
+    const repoUrl = document.querySelector<HTMLInputElement>('[data-repo-url]')
+
+    const updateRepoUrlRequired = () => {
```

Comment:
Oh, nice!  Let's remove the "(required for free themes)" text too and toggle a red * here when it's required